install/0000_30_machine-api-operator_08_machinedisruptionbudget.crd.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: machinedisruptionbudgets.healthchecking.openshift.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.currentHealthy
+    description: The number of healthy machines
+    name: Healthy
+    type: integer
+  - JSONPath: .status.ExpectedMachines
+    description: The total number of machines
+    name: Total
+    type: integer
+  - JSONPath: .status.desiredHealthy
+    description: The desired number of healthy machines
+    name: Desired
+    type: integer
+  group: healthchecking.openshift.io
+  names:
+    kind: MachineDisruptionBudget
+    plural: machinedisruptionbudgets
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: Specification of the desired behavior of the MachineDisruptionBudget.
+          properties:
+            maxUnavailable:
+              description: An deletion is allowed if at most "maxUnavailable" machines
+                selected by "selector" are unavailable after the deletion. For example,
+                one can prevent all voluntary deletions by specifying 0. This is a
+                mutually exclusive setting with "minAvailable".
+              format: int32
+              type: integer
+            minAvailable:
+              description: An deletion of the machine is allowed if at least "minAvailable"
+                machines selected by "selector" will still be available after the
+                deletion. So for example you can prevent all voluntary deletions by
+                specifying "100%".
```

Comment:
"100%" wouldn't be accepted as an integer though right?

Reply:
done